shared-helpers/src/utilities/formKeys.ts

```diff
@@ -56,13 +56,14 @@ export const stateKeys = [
 ]
 
 export const countyKeys = [
+  //Removing options due to existing BayArea Bloom sites
```

Comment:
nit space between `//` and text (did we format this file)
could you add a TODO here to re-add once the county sites have been sunsetted?

Reply:
@lisunshiny I'm not sure I 100% follow your first sentence here, but hoping I addressed both of these now.